lib/components/basic-table/row/__snapshots__/test-row.js.snap

```diff
@@ -1,21 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`BasicTableRow can display rows with expandable content 1`] = `
-<li
-  className="twelve-col basic-table__row basic-table__row--expandable basic-table__row--clickable"
+<tr
```

Comment:
I'm curious as to how this were so wrong from the get go? It's like the snapshots were from a different component, maybe this has the new vanilla code but the old snapshot? Before landing this I'd like this question answered as more changes are being made to https://github.com/juju/juju-gui/pull/3831 so we'll have to keep them in sync.

Reply:
I probably didn't update the snapshots after I copied the components over. There's no CI yet so it wasn't caught before merging.

Reply:
Note that tests do not pass yet, but this makes us closer.